assets/app/scripts/controllers/settings.js

```diff
@@ -26,7 +26,31 @@ angular.module('openshiftConsole')
     ProjectsService
       .get($routeParams.project)
       .then(_.spread(function(project, context) {
-        $scope.project = project;
+        angular.extend($scope, {
+          project: project,
+          show: { display: true, form: false, actions: true },
```

Comment:
Looks like `show.actions` is no longer used.

I suggest one scope var for tracking whether we're showing the form rather than 3 we have to keep in sync.